src/libNode/Node.cpp

```diff
@@ -2089,6 +2089,46 @@ std::string Node::GetActionString(Action action) const {
   return true;
 }
 
+/*static*/ bool Node::GetDSLeaderPubkey(const BlockLink& lastBlockLink,
```

Comment:
The functionality of this function is the same as GetDSLeaderPeer, can combine into one? Or let one call the other?

Reply:
GetDSLeaderPeer return a peer object and not pubkey